frontend/services/gui/src/GUI_Service.cpp

```diff
@@ -389,6 +392,14 @@ namespace frontend {
     }
 
 
+    void GUI_Service::resource_provide_gui_render() {
+
+        if (this->m_gui != nullptr) {
+            this->m_gui->DrawUiToScreen();
+        }
+    }
```

Comment:
I believe in the long run we want such a GUI-Render callback to be called/registered by Image Presentation. Question is if this current callback hack can stay or should be reworked?